### PR TITLE
Send output events to mongo

### PR DESCRIPTION
**Pull request by AntoinePr**
Send output events to the mongo db
It shouldn't raise any error in case the output class has not "output" attribute now.

### CODE REVIEW

File: luigi/task.py [hunk 1]
```diff
@@ -354,6 +355,9 @@ def complete(self):
         However, you may freely override this method with custom logic.
```
Comment (jeremykarn) [inline luigi/task.py]: One thing to note, is this comment.  It's possible that some tasks override this method and so we won't capture their output.  I don't think we do it very often (or maybe at all), so might not be a problem.

File: luigi/task.py [hunk 2]
```diff
@@ -354,6 +355,9 @@ def complete(self):
...
         """
         outputs = flatten(self.output())
+        for output in outputs:
+            output_path = output.path if hasattr(output, 'path') else output_path = ''
```
Comment (raphaelhamonnais) [inline luigi/task.py]: 👍 
Do you think we should send something rather than `' '` or do we just care about the `path` attribute?

Comment (raphaelhamonnais) [reply luigi/task.py]: Hum I think this is a good idea! What makes more sense to you @AntoinePr for Michelada to consider as output, the `table` I would say?

Is there Michelada jobs that have a `RedshiftTarget` as input? Maybe we should look at implementing the same `if has path attribute` logic if this is the case?

**Comment by AntoinePr** [comment]
@antonipp @jeremykarn @raphaelhamonnais I'd like to bring this back to life to go forward with our project :)
do the light I made still fit you ? Can I merge ?

**Comment by jeremykarn** [comment]
I'm fine with merging.  Although lets not do it today. :)

**Pull request merged**